modules/relative-urls.php

```diff
@@ -14,7 +14,7 @@
  * add_theme_support('soil-relative-urls');
  */
  
-if (is_admin() || preg_match('/sitemap(_index)?\.xml/', $_SERVER['REQUEST_URI']) || in_array($GLOBALS['pagenow'], ['wp-login.php', 'wp-register.php'])) {
+if (is_admin() || preg_match('/([a-zA-Z0-9_-]*)sitemap(_index)?(\d*)\.xml/', $_SERVER['REQUEST_URI']) || in_array($GLOBALS['pagenow'], ['wp-login.php', 'wp-register.php'])) {
```

Comment:
thanks for help 
i am beginner in git and make mistake with this PR so close and create new PR